src/commands/lightning/preview/app.ts

```diff
@@ -170,16 +171,39 @@ export default class LightningPreviewApp extends SfCommand<void> {
       logger.debug(`App Id is ${appId}`);
     }
 
+    logger.debug('Determining the next available port for Local Dev Server');
+    const serverPort = PreviewUtils.getNextAvailablePort();
```

Comment:
Staging the server port here, effectively takes away the user's ability to configure or keep their own port, [here](https://github.com/salesforcecli/plugin-lightning-dev/pull/72/files#diff-a4a21c3afa081450784063b77a94e1912b0ba8002b853cf9ff77580cf3fa8674R78). We should probably move the existing persisted-config value logic into here, or maybe rename `PreviewUtils.getNextAvailablePort()` to `PreviewUtils.getPort()`, and add the logic into there.

Reply:
oh good catch, I'll update